schema/samcli.json

```diff
@@ -53,9 +53,128 @@
               "type": "object",
-              "properties": {}
+              "properties": {
+                "hook_name": {
+                  "title": "hook_name",
+                  "type": "string",
+                  "description": "Hook package id to extend AWS SAM CLI commands functionality. \n\nExample: `terraform` to extend AWS SAM CLI commands functionality to support terraform applications. \n\nAvailable Hook Names: ['terraform']"
+                },
+                "skip_prepare_infra": {
+                  "title": "skip_prepare_infra",
+                  "type": "boolean",
+                  "description": "Skip preparation stage when there are no infrastructure changes. Only used in conjunction with --hook-name."
+                },
+                "use_container": {
+                  "title": "use_container",
+                  "type": "boolean",
+                  "description": "Build functions within an AWS Lambda-like container."
+                },
+                "container_env_var": {
+                  "title": "container_env_var",
+                  "type": "string",
+                  "description": "Environment variables to be passed into build containersResource format (FuncName.VarName=Value) or Global format (VarName=Value).\n\nExample: --container-env-var Func1.VAR1=value1 --container-env-var VAR2=value2"
```

Comment:
Is there supposed to be a `\n` between `containers` and `Resources`?

Reply:
There likely should be, but it oddly isn't in the original help text itself. Will make that fix 👍